### ISSUE TITLE
Is it possible to disable gesture to close the slide menu?

### ISSUE THREAD

**Issue by winseyli** [dekatotoro/SlideMenuControllerSwift#111]
As captioned. Thank you for your good work. It's nice!

**Comment by ahmedsafadii** [comment]
yes , drag the gesture as action to your viewController and put inside it

// close
self.slideMenuController()?.closeLeft()
self.slideMenuController()?.closeRight()

**Comment by coderwsli** [comment]
Hello! Thanks for your reply.
I try your solution. I added the swipe gesture recognizer to the view controller and bind the action.
But I got the following log while the slide menu is still pulled out/closed when I do that.

Unbalanced calls to begin/end appearance transitions for <testapp.SlideMenuViewController: 0x14d71430>.

Thanks!

**Comment by ahmedsafadii** [comment]
i assume that you added a View Controller initially

try to used initWithNibName:nil bundle:nil instead of init.

**Comment by ahmedsafadii** [comment]
or if you use View Controller try put this in viewDidLoad

tabBarController.presentModalViewController(ControllerName, animated: false)
